translations/monero-core_de.ts

```diff
@@ -2116,7 +2134,7 @@ Description: </source>
     <message>
         <location filename="../main.qml" line="744"/>
         <source>Balance (syncing)</source>
-        <translation>Guthaben (synchronisierend)</translation>
+        <translation>Guthaben (Synchronisation)</translation>
```

Comment:
Hier ist die alte Übersetzung besser, was meinst Du?

Reply:
'synchronisierend' würde bedeuten, es handelte sich um ein Guthaben mit dem Typ 'synchronisierend', was aber falsch ist, da es um den Status geht. 'Synchronisation' ist daher richtig, da es den Status des Guthabens anzeigt, nämlich dass es sich momentan synchronisiert.

Reply:
> es handelte sich um ein Guthaben mit dem Typ 

Was ist der angesprochene `Typ`? Worin liegt der Unterschied zum `Status`?
`Guthaben (Synchronisation)` hört sich jedenfalls sehr komisch an 🙈

Reply:
Synchronisierendes Guthaben wäre Guthaben, das sich selbst synchronisiert. Aber es gibt nur ein Guthaben (und nicht verschiedene Typen) und das (der Zustand) ist in Synchronisation.

Möglich wäre noch 'synchronisieren', aber 'synchronisierend' ist falsch.

Reply:
+1 für `synchronisieren`

Reply:
Inwiefern ist dann das englische `Balance (syncing)` korrekt?

> Synchronisierendes Guthaben wäre Guthaben, das sich selbst synchronisiert

Ist es denn nicht auch genau so? Das Guthaben gleicht sich doch mit dem aus der Blockchain entnommenen ab.

Reply:
'Balance (syncing)' bedeutet: Balance is syncing
'Guthaben (synchronsierend)' bedeutet: Guthaben ist synchronisierend

Das Guthaben ist aber nicht synchronisierend, sondern einfach nur ein Guthaben, das sich jedoch in Synchronisation befindet. Daher ist 'Synchronisation' das korrekte Wort.

Reply:
Dann lass' uns auf `synchronisieren` einigen. Das hört sich für mich zwar auch ziemlich seltsam an, aber damit können wir glaube ich alle leben.

Reply:
'synchronisiert' wäre auch noch eine Möglichkeit, das habe ich ganz vergessen. Bleibt aber die Gefahr, dass dann jemand denkt, es wäre bereits fertig synchronisiert.

Am besten wäre eigentlich Guthaben (in Synchronisation) oder Guthaben (synchronisiert sich).

Was hält ihr davon? Von der Länge würde es passen.

Reply:
> Guthaben (synchronisiert sich)

😻